docs/intro_tutorial.rst

```diff
@@ -452,3 +452,153 @@ satisfying
+    >>> meas = [meas_1, meas_2]
+    >>> is_povm(meas)
+    True
+
+We may also use the :code:`random_povm` function from :code:`toqito`, and can verify that a
+randomly generated set satisfies the criteria for being a POVM set.
+
+    >>> from toqito.measurement_props import is_povm
+    >>> from toqito.rand import random_povm
+    >>> import numpy as np
+    >>> dim, num_inputs, num_outputs = 2, 2, 2
+    >>> measurements = random_povm(dim, num_inputs, num_outputs)
+    >>> is_povm([measurements[:, :, 0, 0], measurements[:, :, 0, 1]])
+    True
+
+Alternatively, the following matrices
+
+.. math::
+    M_0 =
+    \begin{pmatrix}
+        1 & 2 \\
+        3 & 4
+    \end{pmatrix}
+    \quad \text{and} \quad
+    M_1 =
+    \begin{pmatrix}
+        5 & 6 \\
+        7 & 8
+    \end{pmatrix},
+
+do not constitute a POVM set.
+
+    >>> from toqito.measurement_props import is_povm
+    >>> import numpy as np
+    >>> non_meas_1 = np.array([[1, 2], [3, 4]])
+    >>> non_meas_2 = np.array([[5, 6], [7, 8]])
+    >>> non_meas = [non_meas_1, non_meas_2]
+    >>> is_povm(non_meas)
+    False
+
+Consider the following state:
+
+.. math::
+    u = \frac{1}{\sqrt{3}} e_0 + \sqrt{\frac{2}{3}} e_1
+
+where we define :math:`u u^* = \rho \in \text{D}(\mathcal{X})`.
+
+Define measurement operators
+
+.. math::
+    P_0 = e_0 e_0^* \quad \text{and} \quad P_1 = e_1 e_1^*.
+
+.. code-block:: python
+
+    >>> from toqito.states import basis
+    >>> from toqito.measurement_ops import measure
+    >>> import numpy as np
+    >>> e_0, e_1 = basis(2, 0), basis(2, 1)
+    >>>
+    >>> u = 1/np.sqrt(3) * e_0 + np.sqrt(2/3) * e_1
+    >>> rho = u @ u.conj().T
+    >>>
+    >>> proj_0 = e_0 @ e_0.conj().T
+    >>> proj_1 = e_1 @ e_1.conj().T
+
+Then the probability of obtaining outcome :math:`0` is given by
+
+.. math::
+    \langle P_0, \rho \rangle = \frac{1}{3}.
+
+.. code-block:: python
+
+    >>> measure(proj_0, rho)
+    0.3333333333333334
+
+Similarly, the probability of obtaining outcome :math:`1` is given by
+
+.. math::
+    \langle P_1, \rho \rangle = \frac{2}{3}.
+
+.. code-block:: python
+
+    >>> measure(proj_1, rho)
+    0.6666666666666666
+
+Consider the collection of trine states.
+
+.. math::
+    u_0 = |0\rangle, \quad
+    u_1 = -\frac{1}{2}\left(|0\rangle + \sqrt{3}|1\rangle\right), \quad \text{and} \quad
+    u_2 = -\frac{1}{2}\left(|0\rangle - \sqrt{3}|1\rangle\right).
+
+.. code-block:: python
+
+    >>> from toqito.states import trine
+    >>> from toqito.measurements import pretty_good_measurement
+    >>>
+    >>> states = trine()
+    >>> probs = [1 / 3, 1 / 3, 1 / 3]
+    >>> pgm = pretty_good_measurement(states, probs)
+    >>> pgm
+    [array([[0.66666667, 0.        ],
+           [0.        , 0.        ]]), array([[0.16666667, 0.28867513],
+           [0.28867513, 0.5       ]]), array([[ 0.16666667, -0.28867513],
+           [-0.28867513,  0.5       ]])]
+
+
+Consider the collection of trine states.
```

Comment:
We might want to have a different lead-in sentence here (as we already say "Consider the collection of trine states." up above. Maybe something like:

```
Similarly, we can consider so-called "pretty bad measurement" on the set of trine states
```

We should also provide a reference here to the "pretty bad measurement" paper (which is in the `refs.bib` file already). Feel free to modify/tweak/add to the sentence above as well.